src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -319,6 +319,58 @@ protected virtual JToken SendToAddress(JArray _params)
             return SignAndRelay(snapshot, tx);
         }
 
+        [RpcMethod]
+        protected virtual JToken CancelTransaction(JArray _params)
+        {
+            CheckWallet();
+            var txid = UInt256.Parse(_params[0].AsString());
+            TransactionState state = NativeContract.Ledger.GetTransactionState(system.StoreView, txid);
+            if (state != null)
+            {
+                throw new RpcException(32700, "This tx is already confirmed, can't be cancelled.");
+            }
+
+            var conflict = new TransactionAttribute[] { new Conflicts() { Hash = txid } };
+            Signer[] signers = _params.Count >= 2 ? ((JArray)_params[1]).Select(j => new Signer() { Account = UInt160.Parse(j.AsString()), Scopes = WitnessScope.CalledByEntry }).ToArray() : Array.Empty<Signer>();
```

Comment:
```suggestion
            Signer[] signers = _params.Count >= 2 ? ((JArray)_params[1]).Select(j => new Signer() { Account = AddressToScriptHash(j.AsString(), system.Settings.AddressVersion), Scopes = WitnessScope.CalledByEntry }).ToArray() : Array.Empty<Signer>();
```
To accept both UInt160 or Base58 Address